api/src/messages/model/custom-validators/message-length.ts

```diff
@@ -0,0 +1,30 @@
+import {
+    registerDecorator,
+    ValidationArguments
+} from "class-validator";
+
+function MessageLengthCustomValidator(
+    property: string,
+    minLength: number,
+    maxLength: number,
+) {
+    return function (object: Object, propertyName: string) {
+        registerDecorator({
+            name: "messageLengthValidator",
+            target: object.constructor,
+            propertyName: propertyName,
+            validator: {
+                validate(value: string, args: ValidationArguments) {
+                    const propertyValue = args.object[property];
```

Comment:
Old name is better, cause this is meaningless, which property?

"relatedProperty" is good name.